Scripts/gen_CosmicStage2.sh

```diff
@@ -82,10 +87,16 @@ if [[ ${NJOBS} == 0  || ${NEVTS} == 0 ]]; then
   exit_abnormal
 fi
 
+if [[ ${LOW} != "Low" && ${LOW} != "All" ]]; then
```

Comment:
The capitalization (low vs Low) varies inside this script and across all these scripts.  This could be confusing.  Since All is capitalized I recommend using Low everywhere.

Reply:
The original code had "--low Low" option.
I've added "--low All".
Do you want to change it to "--Low Low"?